spec/site_prism/page_objects/page/organisations/list_access_requests_page.rb

```diff
@@ -10,7 +10,7 @@ class ListAccessRequestsPage < PageObjects::Base
           element :request_date, "[data-qa=\"access-request__request_date\"]"
           element :requester, "[data-qa=\"access-request__requester\"]"
           element :recipient, "[data-qa=\"access-request__recipient\"]"
-          element :approve, "[data-qa=\"access-request__approve\"]"
+          element :view_request, "[data-qa=\"access-request__confirm\"]"
```

Comment:
Is there something more appropriate for element name?


Everything seems to fall into that pattern

` element :xxx, "[data-qa=\"access-request__xxx\"]"`

Not too fussed

Reply:
good point, ill change now 👍